### DISCUSSION TITLE
Hello I want to download the content of an internal table in JSON format through a URL

### DISCUSSION THREAD

**Discussion by tocasoft** [Budibase/budibase#6280] [Help]
Hello
I want to download the content of an internal table in JSON format through a URL. 
Can you help me?

**Comment by melohagan** [comment]
Hi @tocasoft 

The contents of the internal tables are included within an app export.

Alternatively, you can retrieve the rows of a table in JSON format via the API: https://docs.budibase.com/reference/post_tables-tableid-rows-search
